wcfsetup/install/files/ts/WoltLabSuite/Core/Date/Picker.ts

```diff
@@ -698,7 +698,7 @@ const DatePicker = {
             value = "";
           } else {
             element.dataset.value = time.toString();
-            const format = isTimeOnly ? "formatTime" : "formatDate" + (isDateTime ? "Time" : "");
+            const format = isTimeOnly ? "formatTime" : isDateTime ? "formatDateTime" : "formatDate";
```

Comment:
Can't we just split this up into the separate calls? The performance is the same, but the viewer no longer barfs internally.